src/jaxsim/rbda/contacts/rigid.py

```diff
@@ -113,6 +91,34 @@ class RigidContacts(ContactModel):
         default_factory=FlatTerrain
```

Comment:
Just to be coherent with the other contact models:

```suggestion
        default_factory=FlatTerrain.build
```